source/operations/monitoring/collect-minio-metrics-using-prometheus.rst

```diff
@@ -81,21 +109,44 @@ The command returns output similar to the following:
 2) Restart Prometheus with the Updated Configuration
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Append the ``scrape_configs`` job generated in the previous step to the configuration file:
-
-.. code-block:: yaml
-   :class: copyable
+Append the desired ``scrape_configs`` job generated in the previous step to the configuration file:
+
+.. tab-set::
+
+   .. tab-item:: Server metrics
```

Comment:
cluster , since we have node-level metrics and "Server metrics" might imply the wrong granularity.